benefits/static/css/styles.css

```diff
@@ -1,10 +1,29 @@
-/* HTML element overrides */
-
-body {
-  font-weight: 400;
-  font-size: 16px;
-  line-height: 22px;
-  color: #000000;
+:root {
+  --primary-color: #046b99;
+  --primary-color-rgb: 4, 107, 153;
+  --bs-blue: var(--primary-color);
+  --bs-primary: var(--primary-color);
+  --bs-primary-rgb: var(--primary-color-rgb);
+  --text-primary-color: #000000;
+  --bs-body-color: var(--text-primary-color);
+  --bs-body-text-align: left;
+  --bs-body-font-weight: 400;
+  --bs-body-font-size: 16px;
+  --bs-font-sans-serif: Roboto, system-ui, -apple-system, "Segoe UI", sans-serif,
+    "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+  --bs-body-font-family: var(--bs-font-sans-serif);
+  --dark-color: #212121;
+  --footer-background-color: var(--dark-color);
+  --footer-link-color: #73b3e7;
+  --footer-link-hover-color: #9b74d7;
+  --footer-mobile-underline-color: var(--bs-white);
+  --hover-color: #2f4c65;
```

Comment:
What is this `hover-color` for? Can we add some context like the other ones e.g. `footer-link-hover-color`

Reply:
It's for all links and buttons. I'm also following color name conventions from the styleguide itself too: 
<img width="910" alt="image" src="https://user-images.githubusercontent.com/3673236/193363650-f7ad5290-41f4-4996-9602-a9acc2e6d996.png">

Reply:
Got it, thanks!